frontend/src/metabase/query_builder/components/data-search/SearchResultItem.jsx

```diff
@@ -0,0 +1,110 @@
+import styled from "styled-components";
+import React from "react";
+import PropTypes from "prop-types";
+
+import Icon from "metabase/components/Icon";
+import Text from "metabase/components/type/Text";
+
+import Schema from "metabase/entities/schemas";
+import Database from "metabase/entities/databases";
+
+import { color, lighten } from "metabase/lib/colors";
+
+export function SearchResultItem({ item, onSelect }) {
+  const handleClick = () => onSelect(item);
+
+  return (
+    <SearchResultItemRoot onClick={handleClick}>
+      <IconWrapper>
+        <Icon name="table" size={26} />
```

Comment:
```suggestion
        <Icon name="table2" size={22} />
```
Just for consistency's sake, I'd switch this out to our other table icon to match the other items in this picker. I'd also consider making the icons slightly smaller. Right now they look a bit large to me.